docs/cli.md

```diff
@@ -0,0 +1,71 @@
+# CLI Documentation for iXdiagnose
+
+`iXdiagnose` is a command-line tool designed to generate debug information on TrueNAS systems. This documentation provides an overview of the available commands, options, and their usage.
```

Comment:
this doesn't need to be camel case since the binary on disk is literally `ixdiagnose`